index.js

```diff
@@ -107,10 +107,10 @@ const users = [
 ]
 
 function printNameAndPhones(arr) {
-  arr.forEach(function(element, idx) {
-    console.log(arr[idx].name)
-    console.log(`Cell: ${arr[idx].phones.cell}`)
-    console.log(`Office: ${arr[idx].phones.office}`)
+  arr.forEach(function(element) {
+    console.log(element.name)
```

Comment:
I love the refactoring here!
Not only does your previous commit show that you have a good understanding of callbacks, by including the parameters that you were using, it shows that you have a good understanding of how JS handles functions and their parameters!